{{cookiecutter.project_slug}}/apps/pages/wagtail_hooks.py

```diff
@@ -0,0 +1,10 @@
+from wagtail.core import hooks
+
+
+@hooks.register("register_icons")
+def register_fontawesome_icons(icons):
+    # Find more icons https://github.com/FortAwesome/Font-Awesome/tree/master/svgs
+    # or https://github.com/allcaps/wagtail-font-awesome-svg/
+    return icons + [
+        "wagtailfontawesomesvg/solid/share.svg",
```

Comment:
Seems overkill for just one icon! But I guess it's good for cases where it's easy to just pick an icon from the much larger set.

Reply:
Yep. This also makes things *significantly* easier for us to add custom icons if we need to.  Designers could even make new icons for each of our block types 😁

Reply:
Hm. Thinking about it... I wonder if it would make sense – seeing as how we'll end up with this pattern in *all* the projects eventually when they get to this version of wagtail – to put the list of SVG icons into a more obvious location? E.g. `settings.base.WAGTAILADMIN_EXTRA_SVG_ICONS` or something?

Then have consistently across all projects rather than needing to let everyone know they need to go to `apps/pages/wagtail_hooks.py` to add them...

Reply:
Not 100% sure - if you're editing code to use an icon, editing another code (rather than settings) file doesn't seem too bad?

I'm leaving it out, if you want to go for it - then it probably needs to be done across the current projects which already have this implemented (fun!).